package.json

```diff
@@ -38,6 +39,7 @@
     "url": "https://github.com/tgriesser"
   },
   "dependencies": {
+    "@oclif/command": "^1.5.19",
```

Comment:
I know its another dep but I don't think it would be a good idea to roll our own here.